doc/user/layouts/shortcodes/self-managed/aws-terraform-upgrade-notes.html

```diff
@@ -0,0 +1,16 @@
+If upgrading from a deployment that was set up using an earlier version of the
+Terraform modules, see the [Upgrade Notes] for any considerations that might
+apply when using an updated Terraform modules to your existing deployments.
+
+| Terraform version | Notable changes |
```

Comment:
A placeholder until we have release notes per https://materializeinc.slack.com/archives/C07PN7KSB0T/p1742322577770229?thread_ts=1742314177.153309&cid=C07PN7KSB0T.

Once we have that, will have this table include "See the Release notes" or something.